docs/content/users/quickstart.md

```diff
@@ -341,7 +341,76 @@ Here's a quickstart instructions for a number of different environments:
     
     This way we only need to change the value of `DB_CONNECTION` to `ddev` in the `.env` to work with the `db` service.
     This is very handy if you have a local database installed and you want to switch between the connections faster by changing only one variable in `.env`
+    
+    ### Using Vite
+    The default Laravel Vite template will not work with DDEV. To make DDEV and Vite play nicely a few additional steps are required.
+
+    #### Setup a Vite server
+    Install the ddev-viteserve addon.
+    ```bash
+    ddev get torenware/ddev-viteserve
+    ```
+    In `.ddev/.env` change `VITE_PROJECT_DIR=frontend` to `VITE_PROJECT_DIR=.`
```

Comment:
```suggestion
    In `.ddev/.env`, change `VITE_PROJECT_DIR=frontend` to `VITE_PROJECT_DIR=.`
```

Reply:
Legend, thanks for the suggestions, I went ahead and accepted them all :D

Reply:
Thanks for your patience and diligence!